docs/source/modifying-messages.rst

```diff
@@ -0,0 +1,390 @@
+====================
+Message manipulation
+====================
+
+To create malformed messages tlsfuzzer supports modifying them before sending.
+You can apply the modifiers on generator nodes, the ones that send messages
+to the other side.
+
+Custom message generation
+=========================
+
+Tlsfuzzer provides support for sending arbitrary messages over established
+connections.
+It provides two nodes to achieve it: one to send message unencrypted and
+one to send it using current connection status.
+
+Creating unencrypted messages
+-----------------------------
+
+To send a record with a specific payload and type, irrespective of
```

Comment:
irrespective -> regardless

Reply:
I think irrespective works better here, it's ignoring the status, not considering it unimportant

(yes, they are synonyms but I prefer the existing one here)